crawler/block.py

```diff
@@ -129,14 +129,32 @@ def play_events_end(self, cursor):
                             """, {'chain_id': self.chain_id,
                                   'draft_id': draft.draft_id})
                     draft.save(cursor)
+                if ev['type'] == 'stake_unlock':
+                    recp = models.Account(self.chain_id,
+                                          ev['attr']['address'].strip('"'),
+                                          cursor)
+                    recp.balance += int(ev['attr']['amount'].strip('"'))
```

Comment:
In order to handle locked and unlocked stakes, an account should have at least two fields: either total stake and unlocked stake, total stake and locked stake, or locked and unlocked stake. And these fields should be managed when staking and unlocking take places. Either way, balance should NOT be changed on this event.

Reply:
This issue is resolved with https://github.com/amolabs/explorer2/pull/12/commits/7adfe63da85b3821e16bfb1d89b4a72548226363